railties/lib/rails/generators.rb

```diff
@@ -56,6 +56,8 @@ module Generators
         force_plural: false,
         helper: true,
         integration_tool: nil,
+        javascripts: true,
+        javascript_engine: :js,
```

Comment:
Does this default mean ES5 or ES6 support? I think we'd need the latter.

Reply:
Good point. Given `javascript_engine: :es6` exists, `:js` may just be ES5.

Reply:
Just tested generation with the Rails 6 beta. Generated ES6. I think we're safe to keep this revert/commit/PR as is. Though, the tests seem to think otherwise.